pages/collection-list.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: collection-list
+template-title: Collection list
+template-description: List of collection with metadata, search bar, filter, user avatars, and tags. 
+title: Requirements
+description: In order to be certified as a PRP, you need to satisfy all of the requirements. For each requirement, submit proper evidence or responses. If you have questions, please reach out to your assigned Certifying Agent. 
+source-domain: PRP intake questionnaire
+source-url: http://cbprs.org/wp-content/uploads/2020/08/PRP-Intake-Questionnaire.pdf
+figma:
+collection:
+   - title: "1: Implement an information security policy"
+     items:
+       - Comment by Erika Spoon
+       - 3 hours ago
+     image-source: https://images.unsplash.com/photo-1597223557154-721c1cecc4b0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80
+     status: Not Started
+     status-color: bg-primary-lighter
+   - title: "2: Physical, technical and administrative safeguards"
+     items:
+       - Status changed by Ben Cunningham
+       - 4 days ago
+     image-source: https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80
+     status: Revision Needed
+     status-color: bg-secondary-light
+   - title: "3: Employee awareness"
+     items:
+       - Files Uploaded by Jamie Lee
+       - 1 hour ago
+     image-source: https://images.unsplash.com/photo-1542305983-c4100e4b8cd2?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=881&q=80
```

Comment:
Let's pull down photos and make sure they're included in our version control, not referring to external sites.